### ISSUE TITLE
New category

### ISSUE THREAD

**Issue by uliagilak** [graphprotocol/mission-control-curator#354]
Please add Side chains category, 2nd layer of ethereum (or other blockchains)

**Comment by yanivtal** [comment]
Yes we've gotten this request. Sounds like more subcategories of Scaling are warranted. Thanks!
